.github/workflows/rust.yaml

```diff
@@ -36,3 +36,33 @@ jobs:
           pull: true
           push: true
           tags: zimg/rust:1.60.0,zimg/rust:1.60,zimg/rust:edge
+
+  build-github-actions-images:
```

Comment:
```suggestion
  build-neondatabase-images:
```

Reply:
Oops, there was the wrong dir for the image.
The idea is that we are building new image (rust-actions-runner) and using it for all builds in github actions.
It has different base (summerwind/actions-runner), so we could use it directly without needs of intermediate container.